lint/clang_tidy.bzl

```diff
@@ -310,17 +311,19 @@ def _clang_tidy_aspect_impl(target, ctx):
     compilation_context = target[CcInfo].compilation_context
 
     if ctx.attr._options[LintOptionsInfo].fix:
-        patch, report, exit_code, info = patch_and_report_files(_MNEMONIC, target, ctx)
+        outputs, info = patch_and_output_files(_MNEMONIC, target, ctx)
         if len(files_to_lint) == 0:
-            dummy_successful_lint_action(ctx, report, exit_code, patch)
+            dummy_successful_lint_action(ctx, outputs.human.stdout, outputs.human.exit_code, outputs.patch)
         else:
-            clang_tidy_fix(ctx, compilation_context, ctx.executable, files_to_lint, patch, report, exit_code)
+            clang_tidy_fix(ctx, compilation_context, ctx.executable, files_to_lint, outputs.patch, outputs.human.stdout, outputs.human.exit_code)
     else:
-        report, exit_code, info = report_files(_MNEMONIC, target, ctx)
+        outputs, info = output_files(_MNEMONIC, target, ctx)
         if len(files_to_lint) == 0:
-            dummy_successful_lint_action(ctx, report, exit_code)
+            dummy_successful_lint_action(ctx, outputs.human.stdout, outputs.human.exit_code)
         else:
-            clang_tidy_action(ctx, compilation_context, ctx.executable, files_to_lint, report, exit_code)
+            clang_tidy_action(ctx, compilation_context, ctx.executable, files_to_lint, outputs.human.stdout, outputs.human.exit_code)
+
+    clang_tidy_action(ctx, compilation_context, ctx.executable, files_to_lint, outputs.machine.stdout, outputs.machine.exit_code)
```

Comment:
same comment for clang as for shellcheck, https://github.com/aspect-build/rules_lint/pull/331/files#r1680423092